sdk/trace/src/main/java/io/opentelemetry/sdk/trace/samplers/package-info.java

```diff
@@ -3,7 +3,31 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/** Samplers for selecting traces that are recorded and exported. */
+/**
+ * This package contains {@link io.opentelemetry.sdk.trace.samplers.Sampler}s for selecting traces
```

Comment:
Does a package-info have to fully qualify the names in the package?

Reply:
yeah. it appears that it does. :(